crates/driver/src/infra/config/mod.rs

```diff
@@ -14,7 +14,7 @@ pub struct Config {
     pub disable_access_list_simulation: bool,
     pub disable_gas_simulation: Option<eth::Gas>,
     pub solvers: Vec<solver::Config>,
-    pub liquidity: liquidity::Config,
+    pub liquidity: LiquidityConfig,
```

Comment:
This isn't great. This `Config` type isn't supposed to know anything about the config files. It's just a type which collects all infra config objects. So this should be `liquidity::Config` like it was before...